draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -237,7 +237,7 @@ Additional Section of the response.  Clients must either use responses
 included in the additional section returned by the recursive resolver
 or perform necessary SVCB, A, and AAAA record resolutions.  DNS
 authoritative servers may attach in-bailiwick SVCB, A, AAAA, and CNAME
-records in the Additional Section to responses for an SVCB query.
+records in the Additional Section to responses for a SVCB query.
```

Comment:
Since you pronounce this "Ess Vee Cee Bee" it should remain "an"

Reply:
@tfpauly, you can also pronounce it "Service Bee," in which case it's "a".  Ah, the linguistic implications of how we read acronyms....

Unless we're going to specify a pronunciation in the draft?  Would have headed off a whole lot of "giff/jiff" nonsense.